demisto_sdk/tests/validate_content_path_test.py

```diff
@@ -31,6 +33,31 @@
 )
 
 
+@pytest.mark.parametrize(
+    "templates_folder, template_name",
+    (
+        ("template_foo", "template_foo.json"),
+        ("template_foo", "template_foo.yml"),
+    ),
+)
+def test_xdrc_template_file_valid(templates_folder: str, template_name: str):
+    _validate(DUMMY_PACK_PATH / XDRC_TEMPLATE_DIR / templates_folder / template_name)
```

Comment:
Both tests can be simplified, instead of passing `template_foo` four times, just set it as a variable 

```suggestion
@pytest.mark.parametrize(
    "suffix",
    ( ".json", ".yml"),
)
def test_xdrc_template_file_valid(templates_folder: str, template_name: str):
    folder = "foo"
    _validate(DUMMY_PACK_PATH / XDRC_TEMPLATE_DIR / folder / f"{folder}.{suffix}")
```